buildpacks/php/tests/fixtures/platform/generator/path_repository/expected_platform_composer.json

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
I think this file might be unused? I see it's referenced by:
https://github.com/heroku/buildpacks-php/blob/b7bbdea3d5a0d04220733b9d13f27ca6344a0e5e/buildpacks/php/src/tests/platform.rs#L51

...but `ComposerLockTestCaseConfig` is only used by `make_platform_json_with_fixtures()`:
https://github.com/heroku/buildpacks-php/blob/b7bbdea3d5a0d04220733b9d13f27ca6344a0e5e/buildpacks/php/src/tests/platform/generator.rs#L14-L25

...and `make_platform_json_with_fixtures()` doesn't appear to be used anywhere?
https://github.com/search?q=repo%3Aheroku%2Fbuildpacks-php%20make_platform_json_with_fixtures&type=code

(Longer term the testing infra in this repo could do with some dead code clean-up, or else the addition of whatever testing feature was supposed to use these)

Reply:
> ...and `make_platform_json_with_fixtures()` doesn't appear to be used anywhere?

Ah Richard pointed out it's `#[test]` via Slack. In which case this is used :-)